Zend/zend_cpuinfo.h

```diff
@@ -126,58 +126,86 @@ ZEND_API int zend_cpu_supports(zend_cpu_feature feature);
  * functions */
 ZEND_NO_SANITIZE_ADDRESS
 static inline int zend_cpu_supports_sse2(void) {
+#if (defined(__APPLE__) && defined(__aarch64__) && defined(__clang_major__) && __clang_major__ >= 17)
```

Comment:
Just checking for `_aarch64__` should be enough and is simpler

Reply:
Although ideally we would use an allowlust rather than a denylist but that requires knowing what cpu we build for for sure and not all compilers set it the same way